src/semantize/typing.nit

```diff
@@ -2448,6 +2460,28 @@ redef class AIssetAttrExpr
 	end
 end
 
+redef class ASafeExpr
+	redef fun accept_typing(v)
+	do
+		var mtype = v.visit_expr(n_expr)
+		if mtype == null then return # Skip error
+
+		self.mtype = mtype
+
+		if mtype isa MNullType then
+			v.modelbuilder.warning(self, "useless-safe", "Warning: safe operator useless on null.")
```

Comment:
```
Error: can't use safe operator on null?
```

If statically this thing is `null` and you're using a safe operator on it, you're doing something wrong?

Reply:
yeah, static calls on null suck. They are too weird to be correct but they have a clear unambiguous semantic so if feel extra pedant to forbid them. They could also naturally occur in dead code or in generated code, especially with type adaptation.

Nevertheless, currently `null.foo` is forbidden. It's coherent to forbid `null?.foo` too I assume.